libs/nestjs-admin/src/admin.service.spec.ts

```diff
@@ -1,18 +1,40 @@
 import { Test, TestingModule } from '@nestjs/testing'
-import AdminSite from './adminSite'
+import { INestApplication, HttpModule } from '@nestjs/common'
+import { getRepositoryToken, TypeOrmModule } from '@nestjs/typeorm'
+import * as request from 'supertest'
+import { Repository } from 'typeorm'
+import { User } from '../../../src/user/user.entity'
+import { AppModule } from '../../../src/app.module'
```

Comment:
Same

Reply:
And this test should be in a `src/tests` folder probably, as you're doing an integration test using entities from the example app (User)